### PR TITLE
Add SSDT-GPI0 generation

### PR DESCRIPTION
**Pull request by dreamwhite**
As per title, I've added successfully the SSDT-GPI0 functionality to SSDTTime. This feature is useful for laptops which needs VoodooI2C kext to make the trackpad work. 
Feel free to contribute :)

### CODE REVIEW

File: SSDTTime.py [hunk 1]
```diff
@@ -698,6 +697,107 @@ def ssdt_pmc(self):
...
+                ssdt += """    External ([[var]], IntObj)\n""".replace("[[var]]", key)
+
+
+            ssdt += """
+    Scope ([[Root]])
+    {
+        Method (_INI, 0, NotSerialized)  // _INI: Initialize
+        {
+            If (_OSI ("Darwin"))
+            { """.replace("[[Root]]", root)
```
Comment (dhinakg) [inline SSDTTime.py:784]: It's probably better to put the entire template into a single multi-line string and use str.format

Comment (dreamwhite) [reply SSDTTime.py:784]: Alright, it can be replaced with f"{root}"

Comment (dreamwhite) [reply SSDTTime.py:784]: I simply copied the code from SSDT-AWAC and adapted for my needs for gpi0

**Comment by dreamwhite** [comment]
@corpnewt any news about this PR? .-.

**Comment by dreamwhite** [comment]
@corpnewt I think that this issue can be closed as GPI0 generation may cause instability in some cases (e.g. on my laptop if I add the generated SSDT-GPI0 it makes my trackpad not work at all)). Maybe someone will expand this PR and create a detection algorithm for the variables from the trackpad _DSM method.

**Pull request closed without merging**